src/OrchardCore.Modules/OrchardCore.Media/GraphQL/MediaFieldQueryObjectType.cs

```diff
@@ -66,4 +98,22 @@ public MediaFieldQueryObjectType()
                 });
         }
     }
+
+    public sealed class MediaFileItemType : ObjectGraphType<MediaFileItem>
+    {
+        public MediaFileItemType()
+        {
+            Field<StringGraphType>("fileName").Description("the file name of the media file item").Resolve(x => x.Source.FileName);
+            Field<StringGraphType>("path").Description("the path of the media file item").Resolve(x => x.Source.Path);
+            Field<StringGraphType>("url").Description("the url name of the media file item").Resolve(x => x.Source.Url);
```

Comment:
Shouldn't this include `MediaText `?

Reply:
Yes, it should be included, thanks for pointing out